### PR TITLE
chore: use compiled cmak for docker build

### LINKED ISSUE #11: Remove KAFKA_MANAGER_VERSION env from Dockerfile
To update the image for a new version, you need to change 2 ENV variables in Dockerfile. The first one is **KAFKA_MANAGER_SOURCE** - a point to git commit(either tag or version). The second is **KAFKA_MANAGER_VERSION** - used only to unzip `sbt build` output archive (archive contains version as part of a name).

Todo: eliminate **KAFKA_MANAGER_VERSION** variable from Dockerfile and .sh scripts. Use only bash utils to get required folders names

Expected behavior: user updates only **KAFKA_MANAGER_SOURCE**  -> docker image is built successfully using `docker build .`  command.

### LINKED ISSUE #22: consider 11-jre-slim as docker base image

### LINKED ISSUE #24: update build scripts to reflect kafka manager repo naming change
Recently, Kafka Manager was renamed to CMAK. Currently, github handles old repo urls  correctly, but this can be changed in a near future. Thus, we should update our build scripts to point on new repo namings.

See https://github.com/FairwayIQ/kafka-manager-docker/commit/01e444a66092fece9375648549a2033632efad87

### LINKED ISSUE #27: use already compiled version from cmar releases instead compilation

### PR DESCRIPTION
**Pull request by eshepelyuk**
Resolves #27
Resolves #24
Resolves #22
Resolves #11

### CODE REVIEW

**Comment by eshepelyuk** [comment]
@hleb-albau plz review

**Comment by eshepelyuk** [comment]
@hleb-albau since the CMAK version didn't change, but the Dockerfile drastically did, I suggest not to override an image at Dockerhub with the same version 3.0.0.5, but instead create a new Dockerhub repository `hlebalbau/cmak`.

**Comment by hleb-albau** [comment]
Hi, thanks for PR, will check it out in 1-2 days

File: README.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ Kafka Manager images come in two flavors:
 docker run -d \
      -p 9000:9000  \
      -e ZK_HOSTS="localhost:2181" \
-     hlebalbau/kafka-manager:stable \
-     -Dpidfile.path=/dev/null
```
Comment (hleb-albau) [inline README.md:22]: Is problem with pidfile fixed in newer versions of cmak?

Comment (eshepelyuk) [reply README.md:22]: I have no idea about `pidfile problem`, but this JVM system property is already passed to CMAK via `Dockerfile` ENTRYPOINT.

I assume there's no need to duplicate it when running a container.

Correct me if I'm wrong.

Comment (hleb-albau) [reply README.md:22]: gotcha, you are right! resolved

**Comment by hleb-albau** [comment]
tested on local pc, seems good!

**Comment by eshepelyuk** [comment]
Thank you.

Do you have plans to release new docker image anytime soon ?

**Comment by hleb-albau** [comment]
Yeah, right now I suggest next plan:
1. release new `latest` version. test it for some time
2. tag current stable with same as `latest` after testing

also, I did a small research. cmak name is so popular as c-make tool. Try to search cmak on dockerhub. May be it's better to leave name as is for now?

**Comment by eshepelyuk** [comment]
it's up to you about naming thing.
from my side I just interested in some fixed tag - not a stable or latest, to be able to use it it my CMAK operator :)

**Comment by hleb-albau** [comment]
> it's up to you about naming thing.
> from my side I just interested in some fixed tag - not a stable or latest, to be able to use it it my CMAK operator :)

latest rebuilt
than you can use full tag(sha) located  [here](https://hub.docker.com/layers/hlebalbau/kafka-manager/latest/images/sha256-1138c4fe077e873e10c614c3e54ad111382d1cbc3aa829cc45ad5d98291ce56f?context=repo).

**Comment by hleb-albau** [comment]
@eshepelyuk 
```
docker pull hlebalbau/kafka-manager@sha256:1138c4fe077e873e10c614c3e54ad111382d1cbc3aa829cc45ad5d98291ce56f
```
example of immutable digest pull

**Comment by eshepelyuk** [comment]
BTW, don't you object that I'm putting links to your project on my [CMAK operator page](https://github.com/eshepelyuk/cmak-operator#internals) ?

**Comment by eshepelyuk** [comment]
> @eshepelyuk
> 
> ```
> docker pull hlebalbau/kafka-manager@sha256:1138c4fe077e873e10c614c3e54ad111382d1cbc3aa829cc45ad5d98291ce56f
> ```
> 
> example of immutable digest pull

yeah this will be the last resort if no strict version will be release :)

**Comment by hleb-albau** [comment]
> BTW, don't you object that I'm putting links to your project on my [CMAK operator page](https://github.com/eshepelyuk/cmak-operator#internals) ?

it's open source, do want you want!

> yeah this will be the last resort if no strict version will be release :)

I can create special tag for you, if you need (beside `latest` and `master-7e7a22e`). just name it.

**Comment by eshepelyuk** [comment]
> I can create special tag for you, if you need (beside `latest` and `master-7e7a22e`). just name it.

Yeah, it would be great. Could you create a tag `3.0.0.5+7e7a22e` ?

**Comment by hleb-albau** [comment]
`+` is illegal char for tag, so I created `hlebalbau/kafka-manager:3.0.0.5-7e7a22e`

**Comment by eshepelyuk** [comment]
Great, thank you very much.

**Pull request merged**